docs/migrations.md

```diff
@@ -0,0 +1,224 @@
+# Django and Wagtail Migrations
+
+Adding or changing fields on Django models, Wagtail page models
+(which are a particular kind of Django model), or StreamField block classes
+will always require a new [Django schema migration](#schema-migrations);
+additionally, changing field names or types on an existing block will require a
+[Django data migration](#data-migrations).
+
+
+## Table of contents
+
+1. [Reference material](#reference-material)
+1. [Schema migrations](#schema-migrations)
+1. [Data migrations](#data-migrations)
+   1. [Wagtail-specific consideration](#wagtail-specific considerations)
+   1. [Utility functions](#utility-functions)
+
+
+## Reference material
+
+The following links may be useful for setting context or diving deeper
+into the concepts presented throughout this page:
+
+- [Django migrations documentation](https://docs.djangoproject.com/en/1.11/topics/migrations/)
+- [Django data migrations documentation](https://docs.djangoproject.com/en/1.11/topics/migrations/#data-migrations)
+- [Wagtail Streamfield migrations documentation](https://docs.wagtail.io/en/v1.13.4/topics/streamfield.html#migrations)
+
+
+## Schema migrations
+
+Any time you add or change a field on a Django model, Wagtail page model
+(which are a particular kind of Django model), or StreamField block classes, a
+[Django schema migration](https://docs.djangoproject.com/en/1.11/topics/migrations)
+will be required.
+
+To automatically generate a schema migration,
+run the following, editing it to give your migration a name
+that briefly describes the change(s) you're making:
+
+```bash
+./cfgov/manage.py makemigrations -n <description_of_changes>
+```
+
```

Comment:
I would recommend adding a link to existing migrations here to better illustrate the naming pattern and what a good name looks like. Something like:

> For examples of good migration names, see this [list of migrations in cfgov-refresh](https://github.com/cfpb/cfgov-refresh/tree/master/cfgov/v1/migrations).

Reply:
Added in 89f23454e89e7befda7ec3cbe69e2b4ca61cd59e